src/Reanimate/Animation.hs

```diff
@@ -25,10 +62,14 @@ data Animation = Animation Duration (Time -> SVG)
 mkAnimation :: Duration -> (Time -> SVG) -> Animation
 mkAnimation = Animation
 
--- | Construct animation with a duration of @1@.
+-- | Construct an animation with a duration of @1@.
 animate :: (Time -> SVG) -> Animation
 animate = Animation 1
 
+-- | Create an animation with provided @duration@, which consists of stationary frame displayed for its entire duration.
+staticFrame :: Duration -> SVG -> Animation
+staticFrame d svg = Animation d (const svg)
```

Comment:
New function added. Not sure if it will be more useful like this, or without duration parameter (duration would be set to 1)

Reply:
I'm not sure, either. I haven't been too consistent with this. Can't decide whether to always use 'setDuration' or to include a duration parameter to functions like this one. I say we add the function as it is and see how it feels.